data/script.js

```diff
@@ -16,8 +16,13 @@ function configure(json) {
 
     $('#local_ssid').text(json.local_ssid);
     //TODO: use the password length to display
-    if(json.remote_mac != "") $('#remote-scad-code-input').attr('value', formatScadCode(json.remote_mac));
     $('#local-scad-code').text(formatScadCode(json.local_mac));
+    if(json.remote_mac != ""){
+        $('#remote-scad-code').text(formatScadCode(json.remote_mac));
+    } else {
+        $('#remote-scad-code').hide();
+    }
+
```

Comment:
This should modify the remote Yo-Yo code form input to contain the remote mac, but also make the input readonly.
```
if (json.remote_mac != ""){
    $('#remote-scad-code-input').attr('value', formatScadCode(json.remote_mac));
    $('#remote-scad-code-input').attr('readonly', '');
}